src/python/pose_format/utils/holistic.py

```diff
@@ -121,3 +122,56 @@ def load_holistic(frames: list, fps: float = 24, width=1000, height=1000, depth=
                                            additional_holistic_config)
 
     return Pose(header, body)
+
+
+def formatted_holistic_pose(width: int, height: int):
+    dimensions = PoseHeaderDimensions(width=width, height=height, depth=1000)
+    header = PoseHeader(version=0.1, dimensions=dimensions, components=holistic_components("XYZC", 10))
+    body = NumPyPoseBody(fps=10,
+                         data=np.zeros(shape=(1, 1, header.total_points(), 3)),
+                         confidence=np.zeros(shape=(1, 1, header.total_points())))
+    pose = Pose(header, body)
+    return pose
+
+
+def load_mediapipe_directory(directory: str, fps: int, width: int, height: int) -> Pose:
+    """
+    :param directory:
+    :param fps:
+    :param width:
+    :param height:
+    :return:
+    """
+
+    frames = load_frames_directory_dict(directory=directory, pattern="(?:^|\D)?(\d+).*?.json")
+
+    def load_mediapipe_frame(frame):
+        def load_landmarks(name, num_points: int):
+            points = [[float(p) for p in r.split(",")] for r in frame[name]["landmarks"]]
+            points = [(ps + [1.0])[:4] for ps in points]  # Add visibility to all points
+            if len(points) == 0:
+                points = [[0, 0, 0, 0] for _ in range(num_points)]
+            return np.array([[x, y, z] for x, y, z, c in points]), np.array([c for x, y, z, c in points])
+        face_data, face_confidence = load_landmarks("face_landmarks", 128)
```

Comment:
ideally, we don't want to hard code the 128 for the face, since they can be 128 (reduced) or 118 (without refinement) or 468 (full) or 478 (with refinement)

Reply:
add a function argument for it